src/fastertransformer/models/bert/bert_gemm.cc

```diff
@@ -45,6 +52,10 @@ int main(int argc, char* argv[])
     printf("  int8_mode: %d \n", int8_mode);
     std::cout << std::endl;
 
+    bool is_append = is_file_exists("gemm_config.in");
```

Comment:
how about make the file path and name configurable? We frequently need to compare perf between different hardware,  with this hard coded value, we need to duplicate our model for different hardwares.

Reply:
FT is using this path by default in multiple places. Will think about a way to handle it.